doc/esil.md

```diff
@@ -24,6 +24,52 @@ peeks and pokes, spawning a syscall, etc.
 
     [0x00000000]> e asm.esil = true
 
+Running ESIL
+============
+
+In visual mode, `V`, one can iterate through the instructions via the `s` (step) key
```

Comment:
imho is better to document `aes` or `ae` which evaluates and steps from the shell without going into visual because visual cant be scripted that easily. also, if you want to document this, i think is better to make a PR to the book instead. this file aims to be just a very basic reference guide to the esil syntax